ibis/expr/datatypes/__init__.py

```diff
@@ -0,0 +1,13 @@
+# flake8: noqa
```

Comment:
Can you mark the individual lines instead, with their specific ignore codes? `flake8: noqa` disables any and all linting for the file.